sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysis.java

```diff
@@ -79,6 +79,32 @@ private Runner(SensitivityAnalysisProvider provider) {
             return runAsync(network, factorsProvider, contingenciesProvider, SensitivityAnalysisParameters.load());
         }
 
+        public CompletableFuture<SensitivityAnalysisResult> runAsync(Network network,
+                                                                     String workingStateId,
+                                                                     SensitivityAnalysisParameters parameters,
+                                                                     SensitivityFactorsProvider factorsProvider,
```

Comment:
Please keep the same order for the parameters. You also have to update the `runAsync` with all the parameters to assert that the contingenciesProvider is not null.

**Don't forget to add the `breaking change` label.**